ui/render/sampling.py

```diff
@@ -73,8 +80,8 @@ def draw(self, context):
                     if config.using_out_of_core():
                         col.prop(config, "out_of_core_supersampling")
             elif sampler == "METROPOLIS":
-                if denoiser.enabled and denoiser.type == "BCD":
-                    layout.label(text="Can lead to artifacts in the denoiser!", icon=icons.WARNING)
+                #if denoiser.enabled and denoiser.type == "BCD":
+                    #layout.label(text="Can lead to artifacts in the denoiser!", icon=icons.WARNING)
```

Comment:
Why is there so much commented-out code?

Also this goes with the BCD denoiser so also needs to be reverted

Reply:
The pull request is merged and I removed the commented code.